### ISSUE TITLE
Initial testing

### ISSUE THREAD

**Issue by gwaygenomics** [cytomining/cytominer-transport#3]
I am trying out the first pass of cytominer-transport. After installing with `python setup.py install`, I tried the following:

```python
from cytominer_transport import to_parquet

example_source = "s3://imaging-platform/projects/2015_07_01_Cell_Health_Vazquez_Cancer_Broad/workspace/analysis/CRISPR_PILOT_B1/SQ00014610/analysis/"
example_objects = ["cells.csv", "cytoplasm.csv", "nuclei.csv"]
example_destination = "test_dir"

to_parquet(source=example_source, destination=example_destination, objects=example_objects)
```

and received the error: 

```python
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-5-a195cc2e3a3b> in <module>
----> 1 to_parquet(source=example_source, destination=example_destination, objects=example_objects)

~/miniconda3/lib/python3.7/site-packages/cytominer_transport-0.1.0-py3.7.egg/cytominer_transport/_to_parquet.py in to_parquet(source, destination, experiment, image, objects, compression, **kwargs)
     69         image.set_index("ImageNumber")
     70     else:
---> 71         raise FileNotFoundError(filename=pathname)
     72 
     73     # Open object CSVs (e.g. Cells.csv, Cytoplasm.csv, Nuclei.csv, etc.)

TypeError: FileNotFoundError() takes no keyword arguments
```

I then updated the following variables:

```python
# Just one well now
example_source = "s3://imaging-platform/projects/2015_07_01_Cell_Health_Vazquez_Cancer_Broad/workspace/analysis/CRISPR_PILOT_B1/SQ00014610/analysis/A01-1/"
# Capitalize to match file system
example_objects = ["Cells.csv", "Cytoplasm.csv", "Nuclei.csv"]
```

And I received the same error

**Comment by 0x00b1** [comment]
Nice, three different issues:

- [ ] FileNotFoundError call error 
- [ ] I don't check for capitalization 
- [ ] FileNotFoundError error shouldn't be called when the source path includes a URI scheme

**Comment by gwaygenomics** [comment]
@0x00b1 can I suggest to be slightly more explicit about links from issues to code fixes?

I think full-fledged pull requests + review will only slow you down at this stage, but at least documenting these fixes with their individual commits would suffice.

E.g. 

- [x] FileNotFoundError call error fixed in 38eeb309c42b8b8dcd8e9ab6bb4b2ba814c7b9f7
- [x] check for capitalization mismatches fixed in b71783bd9a0f214ccf715dac2e1681ec0e40d7d3
- [x] FileNotFoundError error shouldn't be called when the source path includes a URI scheme fixed in c3c272c789187bb51bdddd6eade2649cc6b12fe0

This will help track code decisions and also help me learn your approach :)
